File: CTDK_Python/063/_063.py
n = int(input("Nhap n: "))

tich = 1;
t = n
while(t != 0):
    dv = t % 10
    if dv % 2 != 0:
        tich = tich * dv
    t = t // 10

print("Tich chu so le: ", tich)

File: LTHAM/063/063.cpp
#include <iostream>
using namespace std;

int TichChuSoLe(int);


int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	cout << "Tich cac chu so le la: " << TichChuSoLe(n) << endl;
	return 0;
}

int TichChuSoLe(int nn)
{
	int s = 1;
	int t = nn;
	while (t != 0)
	{
		int dv = t % 10;
		if (dv % 2 == 1)
			s = s * dv;
		t = t / 10;
	}
	return s;
}